generic_relations/tests/test_serializers.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from rest_framework import serializers
+
+from generic_relations.serializers import GenericModelSerializer
+from generic_relations.tests.models import Bookmark, Note
+
+from .test_relations import BookmarkSerializer, NoteSerializer
+
+
+class TestGenericModelSerializer(TestCase):
+    def setUp(self):
+        self.bookmark = Bookmark.objects.create(
+            url='https://www.djangoproject.com/')
+        self.note = Note.objects.create(text='Remember the milk')
+        self.note2 = Note.objects.create(text='Reticulate the splines')
+
+        self.serializer = GenericModelSerializer(
+            {
+                Bookmark: BookmarkSerializer(),
+                Note: NoteSerializer(),
```

Comment:
Do these have to be Serializers or could they be a `HyperlinkedRelatedField` (for example)?